cdc/sink/producer/kafka/kafka.go

```diff
@@ -130,6 +132,7 @@ func (k *kafkaSaramaProducer) SyncBroadcastMessage(ctx context.Context, message
 			Key:       sarama.ByteEncoder(message.Key),
 			Value:     sarama.ByteEncoder(message.Value),
 			Partition: int32(i),
+			Metadata:  message.Metadata,
```

Comment:
Is it necessary to record partition number in metadata

Reply:
Yes.